frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -94,8 +94,18 @@ class WorkspaceListItem extends React.Component {
 
           {// INFO - GB - 2020-10-14 - The  (level - 1) * 20 + 10 calculation is to have the sequence (10, 30, 50, 70, ...)
             // in other words, the margin starts at 10px at level 1 (first child) and increases by 20px at each new level.
+            // If the space is active the border width (5px) is subtracted
             props.level > 0 && (
-              <div style={{ marginLeft: `${(props.level - 1) * 20 + 10}px` }}>&#8735;</div>
+              <div
+                style={{
+                  marginLeft:
+                  `${(props.level - 1) * 20 + 10 -
+                    (props.location.pathname.includes(`${PAGE.WORKSPACE.ROOT}/${props.workspaceId}/`) ? 5 : 0)
+                  }px`
+                }}
```

Comment:
Would it be possible to move this style in a class?

Reply:
is there a way to pass the variables to the class? doesn't it make it more complicated knowing that we even have a conditional in it?

Reply:
You are right. I'll try to think of an improvement for later.

Maybe put 10 and 20 in named constant though?

Reply:
I don't think is needed here, normally I prefer constants, but in this case they are just number without a meaning, like:
const TEN = 10
const TWENTY = 20
`${(props.level - 1) * TWENTY + TEN - ...